### ISSUE TITLE
ASAPooling bug

### ISSUE THREAD

**Issue by peanwang** [pyg-team/pytorch_geometric#3466]
## 🐛 Bug
I met a bug when I use torch_geometric.nn.ASAPooling.
The graph  have edge_attr. so in the ASAPooling source code (below), add_remaining_self_loops function will fill edge_attr with fill_value= 1 (this is int type),  but add_remaining_self_loops function's arguement fill_value only accept (float, tensor, str) type. so it will raise AttributeError("No valid 'fill_value' provided") exception.

```python
    def forward(self, x, edge_index, edge_weight=None, batch=None):
        """"""
        N = x.size(0)

        edge_index, edge_weight = add_remaining_self_loops(
            edge_index, edge_weight, fill_value=1, num_nodes=N)
```
```python
def add_remaining_self_loops(
        edge_index: Tensor, edge_attr: OptTensor = None,
        fill_value: Union[float, Tensor, str] = None,
        num_nodes: Optional[int] = None) -> Tuple[Tensor, OptTensor]:
```

**Comment by rusty1s** [comment]
Thanks for reporting. This is now fixed in master. Let me know if it works for you :)

**Comment by Zhang-Zhiyuan-zzy** [comment]
I also met this bug today. I change the source code to:

*****************************************************************
    def forward(self, x, edge_index, edge_weight=None, batch=None):
        """"""
        N = x.size(0)

        edge_index, edge_weight = add_remaining_self_loops(
            edge_index, edge_weight, fill_value=torch.tensor(1.0, dtype=torch.float), num_nodes=N)

        if batch is None:
            batch = edge_index.new_zeros(x.size(0))

        x = x.unsqueeze(-1) if x.dim() == 1 else x
*******************************************************************
that，I change the fill_value=1 to torch.tensor(1.0, dtype=torch.float).  
It's run successful, I'm not sure if this will change the original design intention.

i'm a student in middle school and learning pytorch, this question might too simple to answer, but i want to get it.

**Comment by rusty1s** [comment]
Yes, this does work as well. Sorry for the inconveniences!
